### PR TITLE
More robust version extraction for standalone mvupdate

### LINKED ISSUE #161: Standalone mvupdate zeigt keine Versionsnr
Eigentlich soll im log die Versionsnr aus der addon.xml stehen,

`May 01 12:48:44 nas python3[11320]: 2020-05-01 12:48:44.060329 ERROR [mvupdate-0.0:StoreMySQL]: Database error: 1305, 1305 (42000): PROCEDURE mediathekview.ftInsertFilm does not exist
May 01 12:48:44 nas python3[11320]: 2020-05-01 12:48:44.063505 ERROR [mvupdate-0.0:StoreMySQL]: Database error: 1305, 1305 (42000): PROCEDURE mediathekview.ftInsertShow does not exist
`

### PR DESCRIPTION
**Pull request by TpmKranz**
Fixes #161

### CODE REVIEW

File: resources/lib/mvupdate.py [hunk 1]
```diff
@@ -391,20 +391,25 @@ class UpdateApp(AppLogger):
     """ The standalone updater application class """
 
     def __init__(self):
+        scriptpath, scriptname = os.path.split(sys.argv[0])
+        storederr = None
+        version = '0.0'
         try:
-            self.mypath = os.path.dirname(sys.argv[0])
-            tree = ET.parse(self.mypath + '/addon.xml')
+            tree = ET.parse(os.path.join(scriptpath, 'addon.xml'))
```
Comment (alex1702) [inline resources/lib/mvupdate.py:398]: Codacy Issue found: [Using xml.etree.ElementTree.parse to parse untrusted XML data is known to be vulnerable to XML attacks. Replace xml.etree.ElementTree.parse with its defusedxml equivalent function or make sure defusedxml.defuse_stdlib() is called](https://app.codacy.com/manual/MediathekView/plugin.video.mediathekview/pullRequest?prid=5459637)

**Comment by alex1702** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MediathekView/plugin.video.mediathekview/pullRequest?prid=5459637&bid=17894684)

File: resources/lib/mvupdate.py (hunk 1 above)
Comment (TpmKranz) [reply resources/lib/mvupdate.py:398]: Wäre mir das mal vor #155 aufgefallen. :(

Comment (codingPF) [reply resources/lib/mvupdate.py:398]: Das ist eine lokale Resource aus dem Filesystem...da wollen wir mal die Kirche im Dorf lassen :-)

**Pull request merged**